modules/utxo-lib/src/bitgo/signature.ts

```diff
@@ -102,6 +103,21 @@ export function getSignatureVerifications<TNumber extends number | bigint>(
     }
   }
 
+  if (
+    parsedScript.scriptType !== 'taprootKeyPathSpend' &&
+    parsedScript.scriptType !== 'taprootScriptPathSpend' &&
```

Comment:
Don't we have an array of taproot narrowed types already? Let's add it if not.

```
if (!isTaprootScript(parsedScript))

Reply:
unfortunately that won't narrow down the type of `parsedScript` enough


```
export function isParsedScriptTypeTaproot(
  t: ParsedScriptType2Of3
): t is 'taprootKeyPathSpend' | 'taprootScriptPathSpend' {
  return t === 'taprootKeyPathSpend' || t === 'taprootScriptPathSpend';
}
```

![image](https://github.com/BitGo/BitGoJS/assets/283533/2352b3bf-81b3-4b4e-adcc-65a76d3f1ad6)